src/components/App.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Header from './header/Header';
+import Footer from './footer/Footer';
+import Character from './characters/Character';
+import List from './List';
+import styles from './App.css';
+import rickAndMortyCharacters from '../data';
+
+const App = () => (
+  <React.Fragment>
```

Comment:
instead of using <React.Fragment> here, it would be totally fine to use <> and </> (slightly shorter to write)